src/cdn/experimental.js

```diff
@@ -17,6 +17,7 @@ import { Instrument as InstrumentErrors } from '../features/jserrors/instrument'
 import { Instrument as InstrumentXhr } from '../features/ajax/instrument'
 import { Instrument as InstrumentSessionTrace } from '../features/session_trace/instrument'
 import { Instrument as InstrumentSessionReplay } from '../features/session_replay/instrument'
+import { Instrument as InstrumentGenericEvents } from '../features/generic_events/instrument'
 // import { Instrument as InstrumentSpa } from '../features/spa/instrument'
 import { Instrument as InstrumentSoftNav } from '../features/soft_navigations/instrument'
 import { Instrument as InstrumentPageAction } from '../features/page_action/instrument'
```

Comment:
Doesn't this need to be removed?

Reply:
If you are referring to PageActions being removed from the CDN entries, IMO that should be done with the re-write of the feature for better consistency of the changes

Reply:
I just noticed you removed it in the pro polyfills loader. Just wondering if maybe one of these is a mistake?

Reply:
@metal-messiah better would be to leave this out then and replace PA with GE in that PR

Reply:
good catch